Assets/MixedRealityToolkit.Services/DiagnosticsSystem/MixedRealityToolkitVisualProfiler.cs

```diff
@@ -544,6 +546,10 @@ private void BuildWindow()
                 usedMemoryText = CreateText("UsedMemoryText", new Vector3(-0.495f, 0.0f, 0.0f), memoryStats, TextAnchor.UpperLeft, textMaterial, memoryUsedColor, usedMemoryString);
                 peakMemoryText = CreateText("PeakMemoryText", new Vector3(0.0f, 0.0f, 0.0f), memoryStats, TextAnchor.UpperCenter, textMaterial, memoryPeakColor, peakMemoryString);
                 limitMemoryText = CreateText("LimitMemoryText", new Vector3(0.495f, 0.0f, 0.0f), memoryStats, TextAnchor.UpperRight, textMaterial, Color.white, limitMemoryString);
+                voiceCommandText = CreateText("VoiceCommandText", new Vector3(-0.53f, -0.7f, 0.0f), memoryStats, TextAnchor.UpperLeft, textMaterial, Color.white, "Say \"Toggle Profiler\"");
```

Comment:
Would it be possible to move the string in the last parameter out to be a variable, like is done for the lines above?

`private static readonly string usedMemoryString = "Used: ";`

Reply:
They're up on line 38 or so

Reply:
Also maybe even elaborate to "Say \"Toggle Profiler\" to show/hide"?

Reply:
@keveleigh @Troy-Ferrell Applied feedback! Thanks!